### DISCUSSION TITLE
如何设置Input组件的placeholder颜色

### DISCUSSION THREAD

**Discussion by mingxin-yang** [ant-design/ant-design#48472] [Q&A] [answered]
如题

**Comment by kiner-tang** [comment, marked as answer]
你这个问题实际上并不是 antd 的问题，而是原生 input 的问题，可以给 input 组件绑定 class 后设置 `::placeholder` 伪类设置样式：
![image](https://github.com/ant-design/ant-design/assets/10286961/da18312a-bbe7-49e9-bea7-13485e4e4daa)
![image](https://github.com/ant-design/ant-design/assets/10286961/5bff0ab1-ca43-4fbc-91a1-e88765c8f980)

https://stackblitz.com/edit/react-jmnjtp?file=demo.tsx,index.css
